docs/argument-constraints.md

```diff
@@ -1,4 +1,4 @@
-# Argument constraints
+f# Argument constraints

```

Comment:
This seems unintentional!

Reply:
What the actual F?

Glad _you_ saw it!